### PR TITLE
generic projectile label override

### PR DESCRIPTION
**Pull request by CMDR-Bill-Doors**
## Additions

GenericLabelOverride field in CombatExtended.AmmoDef. Ammo generalizer will use it for projectile name if present.

## Changes

Translation key for " bullet " in Ammo generalizer, since I stumbled across it.
NoTranslate annotation in AmmoDef's oldDescription, it has been annoying me forever and I happened to come across it.

## Reasoning

Intended for edge cases.

## Alternatives

Revolver shotgun SMG.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] Playtested a colony (a full mag from a machine pistol)

### CODE REVIEW

File: Defs/Ammo/Pistols/45ACP.xml [hunk 1]
```diff
@@ -1,224 +1,225 @@
...
+    <statBases>
+      <MarketValue>0.09</MarketValue>
+    </statBases>
+    <ammoClass>HollowPoint</ammoClass>
+    <cookOffProjectile>Bullet_45ACP_HP</cookOffProjectile>
+  </ThingDef>
+
+  <!-- ================== Projectiles ================== -->
+
+  <ThingDef Class="CombatExtended.AmmoDef" Name="Base45ACPBullet" ParentName="BaseBulletCE" Abstract="true">
```
Comment (N7Huntsman) [inline Defs/Ammo/Pistols/45ACP.xml:100]: Why add `Class="CombatExtended.AmmoDef"` here, seeing as these are projectiles, not ammo items?

Comment (CMDR-Bill-Doors) [reply Defs/Ammo/Pistols/45ACP.xml:100]: label override is a field in ammodef.

**Review by N7Huntsman — APPROVED** [review]
Works.

**Review by MaxDorob — APPROVED** [review]
